tests/e2e/vmgroupbasic_test.go

```diff
@@ -0,0 +1,240 @@
+}
+
+func (s *vgBasicSuite) TearDownTest(c *C) {
+	out, err := admincli.DeleteVMgroup(s.config.EsxHost, vmGroupName)
+	c.Assert(err, IsNil, Commentf(out))
+}
+
+var _ = Suite(&vgBasicSuite{})
+
+// Verify vmgroup for orphaned volumes is specified as N/A
+// 1. Create a vmgroup and add vm-2 to it.
+// 2. Verify vmgroup was successfully created.
+// 3. Create a volume from vm-2.
+// 4. Verify volumes vmgroup from esx.
+// 5. Remove vm-2 from vmgroup.
+// 6. Verify volume is not visible from vm-2.
+// 7. Delete the tenant.
+// 8. Again verify volumes vmgroup is listed as N/A.
+func (s *vgBasicSuite) TestOrphanedVolumesVmgroups(c *C) {
+	misc.LogTestStart(c.TestName())
+	// s.volumeName1 = inputparams.GetUniqueVolumeName(vmGroupName)
+
+	// Create a vmgroup and add vm to it.
+	admincli.CreateVMgroup(s.config.EsxHost, vmGroupName, s.config.DockerHostNames[0], admincliconst.VMHomeDatastore)
+
+	// Verify if vmgroup exists
+	isVmgroupAvailable := admincli.CheckVmgroupAvailability(s.config.EsxHost, vmGroupName)
+	c.Assert(isVmgroupAvailable, Equals, true, Commentf("vmgroup ls command does not lists the vmgroup %s .", vmGroupName))
+
+	// Create a volume from vmgroup's vm
+	out, err := dockercli.CreateVolume(s.config.DockerHosts[0], s.volumeNames[0])
+	c.Assert(err, IsNil, Commentf(out))
+
+	// Check if volume was successfully created
+	isAvailable := verification.CheckVolumeAvailability(s.config.DockerHosts[0], s.volumeNames[0])
+	c.Assert(isAvailable, Equals, true, Commentf("Volume %s is not available after creation", s.volumeNames[0]))
+
+	// Verify if volume exists in test's vmgroup
+	isVolInVmgroup := admincli.CheckVolumeExistInVmgroup(s.config.EsxHost, s.volumeNames[0], vmGroupName)
+	c.Assert(isVolInVmgroup, Equals, true, Commentf("Volume [%s] does not belong to vmgroup [%s]", s.volumeNames[0], vmGroupName))
+
+	admincli.RemoveVMFromVMgroup(s.config.EsxHost, vmGroupName, s.config.DockerHostNames[0])
+
+	// Check if volume was not visible from vm since it does not belong to vmgroup
+	isAvailable = verification.CheckVolumeAvailability(s.config.DockerHosts[0], s.volumeNames[0])
+	c.Assert(isAvailable, Equals, false, Commentf("Unexpected Behavior: Volume %s belonging to "+
+		"vmgroup [%s] is visible from host [%s] which does not belong to the same vmgroup", s.volumeNames[0], vmGroupName, s.config.DockerHosts[0]))
+
+	// Now delete the tenant
+	admincli.DeleteVMgroup(s.config.EsxHost, vmGroupName)
+
+	// Verify vmgroup does not exist
+	isVmgroupAvailable = admincli.CheckVmgroupAvailability(s.config.EsxHost, vmGroupName)
+	c.Assert(isVmgroupAvailable, Equals, false, Commentf("Failed to delete the vmgroup [%s] .", vmGroupName))
+
+	// Verify volume does not belong to any vmgroup
+	isVolInVmgroup = admincli.CheckVolumeExistInVmgroup(s.config.EsxHost, s.volumeNames[0], NotApplicable)
+	c.Assert(isVolInVmgroup, Equals, true, Commentf("Unexpected Behavior: Vmgroup [%s] for volume [%s] is not N/A. ", s.volumeNames[0], vmGroupName))
+
```

Comment:
@govint No, I am not removing the volume here as no vmgroup exist for that volume. Is it ok if I write a util that will remove such orphaned volumes using command: 
``` rm /vmfs/volumes/<datastore>/dockvols/<vmgroup_name>/<volumeName*>```